airbyte-api/src/main/openapi/config.yaml

```diff
@@ -2301,6 +2325,25 @@ paths:
                 $ref: "#/components/schemas/ImportRead"
         "404":
           $ref: "#/components/responses/NotFoundResponse"
+  /v1/internal/set_temporal_workflow_in_attempt:
```

Comment:
Would it make sense to avoid using the word temporal in our API? I know it's an "internal" API, but still it feels like we are leaking an implementation detail. Maybe just `set_workflow_in_attempt` or `set_workflow_metadata_in_attempt`?

Reply:
Continued in child PR here: https://github.com/airbytehq/airbyte/pull/15902/files#r953194463